packages/lexical-react/src/LexicalContentEditable.tsx

```diff
@@ -82,14 +82,15 @@ export function ContentEditable({
       aria-labelledby={ariaLabelledBy}
       aria-multiline={ariaMultiline}
       aria-owns={!isEditable ? undefined : ariaOwns}
+      {...(!isEditable ? {'aria-readonly': true} : null)}
```

Comment:
Nit, I believe the use of undefined should do the same
```suggestion
      aria-readonly={!isEditable ? true : undefined}
```

Reply:
Good point! That is more readable 👍